namespaces/Representor.md

```diff
@@ -0,0 +1,47 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Element)
+
+The Resource represents a Hypermedia Resource, with its available transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `attributes` (object)
+    - `relation` - (string) - An optional relation to this resource, applicable when embedded in another resource.
+- `content` (object)
```

Comment:
I think @smizell had the same point here – and he suggested that this should actually be an array of elements not a hash with elements as values.

Please @smizell correct me if I am wrong (because I think this is easier and it is aligned with what Member Element is at the moment)